source/_docs/drush.md

```diff
@@ -365,10 +365,30 @@ See the [Drush Migrate documentation](https://drupal.org/node/1561820) for detai
 ## Known Limitations
 - Crontab: Currently, there is no way to manage Crontab on Pantheon. If you need a way to set up your own Cron interval, you can use an external cron service such as [Easy Cron](https://www.easycron.com/user/register).
 - The following Drush commands are not supported and will not work on Pantheon sites: `sql-sync-pipe`, `sql-sqlc`, `php-eval`. An alternative to `drush sql-sync` is to use `drush sql-dump` instead.
-- Incorrect `['uri']` in `pantheon.aliases.drushrc.php` file. Drush may fail if the `['uri']` array key has a different domain than what is expected by Drupal, resulting in the following error:
+- Drush may fail if the `['uri']` array key has a different domain than what is expected by Drupal, resulting in the following error:
 
  ```bash
  drush @pantheon.example.live  st
  Drush command terminated abnormally due to an unrecoverable error.       [error]
  ```
- Setting the `--uri` option will not work. To resolve this error, use a [Drush policy file](#drush-alias-strict-control).
+ Setting the `--uri` option will not work. To resolve this error, use a [Drush policy file](#drush-alias-strict-control) or drop the following in `drush/drushrc.php`:
+
+ ```
+   $uri = 'http://usenix.dev';
+   if (isset($_SERVER['PANTHEON_ENVIRONMENT'])) {
+     $uri = 'https://' . $_SERVER['PANTHEON_ENVIRONMENT'] . '.usenix.dev';

```

Comment:
This line will cause problems if the user has not set up DNS for all environments.  Also, do we know if users typically use a DNS wildcard for their dev environments, or whether they add one record per environment?  If the later, then they would need to ensure that they always updated DNS every time they created a new multidev environment.

It might be worth a couple of words of warning here.  Usually folks do not set up their dev environments to redirect (I think -- maybe you have more information on this).  Perhaps the best thing would be to comment out this line, and add a comment that this should be customized iff needed for dev environments.

Finally, I'm not sure that ".dev" is the best example TLD to use in the document.  Some devs set up DNS on their dev machines such that "*.dev" == localhost.  Others use "*.local" for this same purpose, so this is not universal.  I'm not sure what folks do for their Pantheon environments, but I would expect that most of the time they would set up 'test.usenix.org' as a CNAME for their pantheonsite.io domain.